### PR TITLE
Gi 136 home design anpassen

### PR DESCRIPTION
**Pull request by MerlindCPohl**
## Description

Implements the mobile view and the desktop view for the Homescreen.

> **Reminder:** The Jira ticket **must be linked in the PR title**  
> Example: `GI-21: Implement user login flow`

## Jira Ticket
Gi 136 home design anpassen
https://techstudyfinder.atlassian.net/jira/software/projects/GI/boards/1?selectedIssue=GI-136

## Author Checklist

- [X ] Code follows project coding standards and conventions
- [X ] Tests have been written or updated
- [X ] All tests pass locally
- [X ] No sensitive data or secrets are included
- [X ] Documentation (README, comments) has been updated if necessary

## Reviewer Notes

Please verify the following during review:

- The implementation matches the described requirements and Jira ticket
- Code changes are logically correct and maintainable
- No unnecessary dependencies or side effects introduced
- Tests cover critical logic and edge cases
- Naming, structure, and readability meet team standards

## Additional Context (optional)

Add any background information or references that help reviewers understand the changes.

---

_Please ensure all criteria are met before merging._

### CODE REVIEW

**Comment by MerlindCPohl** [comment]
Hi, eigenlitch ist der Button gleich groß, nur die Karte im verhältnis etwas klein, aber das muss so, weil sie nur so wie sie jetzt ist auf allen unterschiedlichen Mobiles passt, sonst ist sie immer für sehr kleine Phones wieder verschoben, oder es sieht auf den längeren Bildschirmen doof aus.
Vielleicht können wir am Donerstag nochmal sprechen und ggf. erstmal auf ein Handy einigen.

File: client/src/components/nav-bar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+      sx={{
+        position: "fixed",
+        bottom: 20,
+        left: "50%",
+        width: "80%",
+        transform: "translateX(-50%)",
+        borderRadius: 50,
+        zIndex: 1000,
+      }}
+      elevation={3}
```
Comment (DamarisLi) [inline client/src/components/nav-bar/NavBar.tsx]:
```suggestion
      elevation={0}
```
Den Schatten vielleicht entfernen, dann sieht es aus wie im Prototyp und die Card ist mehr betont

**Review by DamarisLi — COMMENTED** [review]
Sieht schon echt gut aus soweit, hab nur ein paar kleine Sachen angemerkt!

**Review by DamarisLi — APPROVED** [review]
